pkgs/servers/cookbook/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, pythonPackages, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "cookbook";
+  version = "1.0.0";
+  src = fetchFromGitHub {
+    owner = "strangeglyph";
+    repo = "cookbook";
+    rev = "v${version}";
+    sha256 = "sha256-ylpQweY1uhzk8vINGGNrQpS/jkVnAWeY9YZfD/i3zdE=";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [ flask ruamel-yaml ];
+
+  pythonImportsCheck = [ "flask" "ruamel.yaml" ];
+  doCheck = false;
```

Comment:
Why?

Reply:
I am not sure what the check phase does exactly, but it seems to try to execute the application directly, which then fails without proper setup (some required env variables and a config file). There aren't any proper tests in the project anyway, I'm ashamed to admit

Reply:
This being said, add a comment like `# the package provides no tests`.